### ISSUE TITLE
add a plot for the string angle

### ISSUE THREAD

**Issue by lleroy** [bow-simulation/virtualbow#253]

**Comment by Puujousi** [comment]
Good idea! But dow do you define unambiguously string angle, especially with contact recurves?

**Comment by lleroy** [comment]
I'd say string angle at the nocks, angle between the string and the bow, so indeed including the angle of the recurve
(purpose would be to check that it's not over 90, so it won't risk the string to slip off)

If it's "just" a curve, you could add also have one with the string angle at the the arrow nock, but I do not see much use for that.

for "contact recurves" it does not make much sense, since it would be 0 I suppose. (but I don't make those :-)

**Comment by stfnp** [comment]
Measuring the angle at the nock seems reasonable to me. For recurves that would mean that the angle is zero initially until the string has been lifted off the limb, but I think that's okay.
